at_client/lib/src/client/at_client_impl.dart

```diff
@@ -520,6 +520,13 @@ class AtClientImpl implements AtClient {
         sharedWith: atKey.sharedWith, metadata: atKey.metadata);
   }
 
+  @override
+  Future<dynamic> runBatchCommand(List<VerbBuilder> atVerbBuilders) async {
+    var batchRequests = _getBatchRequests(atVerbBuilders);
+    var batchResponse = await _sendBatch(batchRequests);
+    return batchResponse;
```

Comment:
this must be list of BatchResponse since end user will not be aware of id inside server response